<launch>
	<arg name="frame" default="xsens" />	
	<node pkg="mtig_driver" type="mtig_driver_node" name="mtig_driver" output="screen" />
		<!-- Error parameters /-->		
		<param name="roll_error" value="0.2" />
		<param name="pitch_error" value="0.2" />		
		<param name="yaw_error" value="1.0" />
		<param name="acc_noise" value="0.00015" />
		<param name="gyr_noise" value="0.01" />
	
		<!-- Frame Parameter /-->
		<param name="frame_id" value="$(arg frame)" />				
		
		<param name="orientation_enabled" value="true"/>
		<!-- Orientation Max Freq = 400-->
		<param name="orientation_frequency" value="400"/>

		<param name="gps_enabled" value="true"/>
		<!-- GPS Max Freq = 400-->
		<param name="gps_frequency" value="400"/>

		<param name="temperature_enabled" value="true"/>
		<!-- Temperature Max Freq = 400-->
		<param name="temperature_frequency" value="1"/>

		<param name="acceleration_enabled" value="true"/>
		<!-- Acceleration Max Freq = 400-->
		<param name="acceleration_frequency" value="400"/>

		<param name="pressure_enabled" value="true"/>
		<!-- Pressure Max Freq = 50-->
		<param name="pressure_frequency" value="50"/>

		<param name="magnetic_enabled" value="true"/>
		<!-- Magnetic Max Freq = 400-->
		<param name="magnetic_frequency" value="100"/>
		
		<param name="altitude_enabled" value="true"/>
		<!-- Altitude Max Freq = 400-->
		<param name="altitude_frequency" value="400"/>

		<param name="velocity_enabled" value="true"/>
		<!-- Velocity Max Freq = 400-->
		<param name="velocity_frequency" value="400"/>

		<param name="gyroscope_enabled" value="true"/>
		<!-- Gyroscope Max Freq = 200-->
		<param name="gyroscope_frequency" value="200"/>
</launch>
